docs/source/AdministratorGuide/Systems/DataManagement/dfc.rst

```diff
@@ -0,0 +1,80 @@
+
+FileCatalogHandler
+------------------
+
+All the configuration of the DFC takes place there.
+
+* `DatasetManager`: default `DatasetManager` Manager for the dataset
+* `DefaultUmask`: default `0775` Umask in octal
+* `DirectoryManager`: default `DirectoryLevelTree` Manager for the Directories
+* `DirectoryMetadata`: default `DirectoryMetadata` Manager for the directory metadata
+* `FileManager`: default `FileManager` Manager for the files
+* `FileMetadata`: default `FileMetadata` Manager for the file metadata
+* `GlobalReadAccess`: default `True`. If set to True, anyone can read anything
+* `LFNPFNConvention`: default `Strong`.
+* `ResolvePFN`: default `True`. Deprecated
+* `SecurityManager`: default `NoSecurityManager`. Manager for authentication
+* `SecurityPolicy` : if `SecurityManager = PolicyBasedSecurityManager`, path to the policy to use
+* `SEManager`: default `SEManagerDB`. Managers for the strage elements
+* `UniqueGUID`: default `False`. If `True`, the GUID has to be unique through the namespace
+* `UserGroupManager`: default `UserAndGroupManagerDB`. Managers for groups and users
+* `ValidFileStatus`: default `[AprioriGood,Trash,Removing,Probing]`. Status that are valid for Files
+* `ValidReplicaStatus`: default `[AprioriGoodTrashRemovingProbing]`. Status that are valid for Replicas
+* `VisibleFileStatus`: default `[AprioriGood]`. By default, only files in this status are returned
+* `VisibleReplicaStatus`: default `[AprioriGood]` By default, only replicas in this status are returned
+
+In order to use the LHCb handler you should:
+
+* `FileManager = FileManagerPs`
+* `DirectoryManager = DirectoryClosure`
+* `UniqueGUID = True`
+* `SecurityManager = PolicyBasedSecurityManager`
+* `SecurityPolicy = DIRAC/DataManagementSystem/DB/FileCatalogComponents/SecurityPolicies/VOMSPolicy`
+
+
+Security Manager
+----------------
+
+This manager takes care of the access permissions in the DFC. There are several of them:
+
+* `NoSecurityManager`: offer yourself to whatever treatment the world reserves you
+* `DirectorySecurityManager`: only look at directories for permissions
+* `FullSecurityManager`:
+* `DirectorySecurityManagerWithDelete`: same as `DirectorySecurityManager` but consider the parent's directory write bit for removal
+* `PolicyBasedSecurityManager`: based on plugins. It will evaluate the permissions based on the path, the identity doing the request, and the action itself on a per method bases. Currently, only the `VOMSPolicy` exists in DIRAC.
+
+The `VOMSPolicy` (:py:class:`~DIRAC.DataManagementSystem.DB.FileCatalogComponents.SecurityPolicies.VOMSPolicy.VOMSPolicy`) implements a 3-level posix permission (directory-file-replica), and groups the dirac group using their VOMS roles. Basically, if the owner does not match, the groups are used. But the group doing the request and the one owning the file do not need to be the same: it is enough if they share the same VOMS role.
+
+
+
+
+LFN PFN convention
+------------------
```

Comment:
I think here you should add a section on "What should be installed to have a DFC up and running?"

Reply:
It is above
`
The list of components you need to have installed is:

   * FileCatalogDB: if you want the standard managers, you should use `FileCatalogDB.sql`, but `FilecatalogWithFkAndPsDB.sql` if you want the LHCb ones
   * FileCatalogHandler: just the interface to the DB
`